src/test/kotlin/fi/metatavu/timebank/api/test/functional/data/TestData.kt

```diff
@@ -54,6 +54,7 @@ class TestData {
 
         /**
          * Gets mock Forecast API response to /v4/time_registrations
+         * Used to test synchronizationDeletedTimeEntries -endpoint
```

Comment:
Should the synchronizeDeletedTimeEntries endpoint be merged into the regular synchronization?